### PR TITLE
[DOCS-7390] Search Enterprise 4.0

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: search-enterprise/latest/support/index.md
[the remark this answers is not in the document]
Comment (anxumalo) [reply search-enterprise/latest/support/index.md:1]: Replace this line: `| JDK 11 or OpenJDK 11 | |`

with the following block:
```
| | |
| | Check the [Alfresco Content Services Supported platforms]({% link content-services/latest/support/index.md %}) page for specific versions of the individual components. |
| **Java** | |
| JDK x or OpenJDK x | |
| | |`

**Review by anxumalo — CHANGES_REQUESTED** [review]
Since SE 4.0 is only compatible with Alfresco Content Services 23.1, there are several places that still refer to ACS 7.4 (affects top-level `index.md`, `/install`, and `/support` pages).

File: search-enterprise/latest/support/index.md
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply search-enterprise/latest/support/index.md:1]: done

**Review by anxumalo — COMMENTED** [review]
See inline comments - require technical approval

File: search-enterprise/latest/support/index.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
 | ------- | ----- |
-| Content Services 7.4.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
-| Content Services 7.3.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
-| Content Services 7.2.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
-| Content Services 7.1.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
-| JDK 11 or OpenJDK 11 | |
+| Content Services 23.1.x | Including ActiveMQ, Alfresco Transform Service, and PostgreSQL unless the versions differ from what's supported in ACS. |
+| | Check the [Alfresco Content Services Supported platforms]({% link content-services/latest/support/index.md %}) page for specific versions of the individual components. |
+| **Java** | |
+| JDK x or OpenJDK x | |
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply search-enterprise/latest/support/index.md]: done

**Review by anxumalo — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**